core/src/main/java/org/apache/accumulo/core/file/rfile/RFile.java

```diff
@@ -746,50 +746,45 @@ public boolean hasTop() {
     @Override
     public void next() throws IOException {
```

Comment:
I don't think this method is a problem anymore for the JIT.  There were other methods in Rfile that popped up though, I listed all the ones I found here: https://issues.apache.org/jira/browse/ACCUMULO-4621?jql=project%20%3D%20ACCUMULO